core/src/main/java/org/mobilitydata/gtfsvalidator/table/ClassGraphTableDiscoverer.java

```diff
@@ -0,0 +1,46 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.flogger.FluentLogger;
+import io.github.classgraph.ClassGraph;
+import io.github.classgraph.ClassInfo;
+import io.github.classgraph.ScanResult;
+
+/*
+Discovers GtfsTableDescriptor subclasses in the default table package.
+ */
+public class ClassGraphTableDiscoverer {
```

Comment:
I'm in favor of combing all the classpath discovery code into one impl class, combing Table and Validator scanning in one place.  I'd also move existing Notice discovery code here as well.  That way, all ClassGraph-based discovery code would be in one place, where it's easier to make sure it's being used in a consistent way.  I'd propose calling the combined class ClassGraphDiscovery or something and putting it in the validator/ subpackage.  Thoughts?

Reply:
Good idea, done.

Notice discovery code will be moved to the new ClassGraphDiscovery class in a separate commit.